common/sgx/revocation.h

```diff
@@ -49,6 +49,20 @@ oe_result_t oe_get_revocation_info_from_certs(
     oe_cert_t* intermediate_cert,
     oe_get_revocation_info_args_t* args);
 
+/**
+ * Fetch quote verification collateral from the quote provider given the PCK
+ * certificate and CA certificate.
+ *
+ * Caller is responsbile for freeing the quote verification collateral resources
+ * by calling oe_free_quote_verification_collateral_args().
+ *
+ * @param[in] leaf_cert The PCK certificate.
+ * @param[out] args Quote verification collateral.
+ */
+oe_result_t oe_get_quote_verification_collateral_from_certs(
```

Comment:
recommend inserting _sgx_ in the function name

Reply:
Thanks, changed to oe_get_sgx_quote_verification_collateral_from_certs